code/modules/xenomorph/xenoplant.dm

```diff
@@ -136,6 +143,14 @@
 	///How long should the buff last
 	var/duration = 1 MINUTES
 
+/obj/structure/xeno/plant/plasma_fruit/deconstruct(disassembled = TRUE, mob/living/blame_mob)
+	if(!disassembled && mature)
```

Comment:
Why does this one not trigger burst on uproot like the others, intentional balance thing? Code looks fine otherwise.

Reply:
forgor.